### DISCUSSION TITLE
Change the AABB name to BoundingBox3D

### DISCUSSION THREAD

**Discussion by hidemat** [godotengine/godot-proposals#5167] [3D]
Just a thought. The AABB name might be a little abstract for new comers. BoundingBox3D I feel would make it more obvious what the class is used for just from reading the name. Opinions?

**Comment by Mickeon** [comment]
**AABB** is the 3D equivalent of **Rect2**. t's used in bounding boxes, but it is not the bounding box in and of itself. At its core it's just a rectangular cuboid.

**Comment by Calinou** [comment]
AABB was called Rect3 back in the Godot 2.x era. Its current name (AABB) is intended to make its *axis-aligned* nature explicit.

**Reply by hidemat** [reply]
Understood. I see the reason for its name.
